mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/MqttHighLevelApi.scala

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.mqtt.streaming
+
+import akka.actor.ActorSystem
+import akka.annotation.ApiMayChange
+import akka.stream.scaladsl.{Flow, Tcp}
+import akka.util.ByteString
+import akka.util.JavaDurationConverters._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+
+@ApiMayChange
+trait MqttTransportSettings {
+  def connectionFlow()(implicit system: ActorSystem): Flow[ByteString, ByteString, _]
+}
+
+@ApiMayChange
+final class MqttTcpTransportSettings private (
+    val host: String,
+    val port: Int
+) extends MqttTransportSettings {
```

Comment:
No TLS considerations? I haven't yet connected to a single broker in the wild that doesn't require TLS.

Reply:
That's supported via `MqttTransportSettings` which is just a factory interface for connection flows.